.dockerignore

```diff
@@ -1,9 +1,5 @@
-bin/
-config/
-.git/
-hack/
-.gitignore
-LICENSE
-Makefile
-PROJECT
-README.md
```

Comment:
I just kept the default file in this case because I think it looked cleaner to have an allow-list instead have a block-list and keep updating it whenever a new kind of file is added to the project. Let me know if you prefer to have both and I can add them back.